whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -183,6 +185,7 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
 
         final RdapObject rdapObject = mapCommonNoticesAndPort(searchResult, requestUrl);
         mapCommonLinks(rdapObject, requestUrl);
+        includeRirSearchConformance(rdapObject, requestUrl);
         return mapCommonConformances(rdapObject);
```

Comment:
This conformance should be included if it is a RIR search ?

Reply:
The lira says 
"Always return the "rirSearch1" extension on all responses."

Reply:
If that is the case  we should add all conformance to every  rdap response but we do not. For ex: we add geofeed  Conformance only if it is Ip request . I think we need to be consistent about it.  Let's discuss it with Ed

Reply:
I have been talking to Ed
"rirSearch1" will only appear in the /help
so I've removed it from "includeRirSearchConformance" the rest of the conformance should be there depending if the search is IP or Autnums

Reply:
we are only doing for basic search as the RFC points
so it will not be any conformance for UP or TOP because they are lookup

Reply:
updated the description and commented via chat